tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/arena/ElemPtr.scala

```diff
@@ -0,0 +1,65 @@
+package at.forsyte.apalache.tla.bmcmt.arena
+
+import at.forsyte.apalache.tla.bmcmt.ArenaCell
+import at.forsyte.apalache.tla.lir.TypedPredefs.BuilderExAsTyped
+import at.forsyte.apalache.tla.lir.convenience.tla
+import at.forsyte.apalache.tla.lir.{BoolT1, TlaEx, UID}
+
+/**
+ * An abstract membership pointer.
```

Comment:
I think this doccomment can be improved 🙂
How and where are these membership pointers used?
I think this should at least mention some reference to arenas and this ADR.